tests/requirements.txt

```diff
@@ -2,3 +2,5 @@ pytest >= 5.0.0
 pytest-cov >= 2.6.0
 codecov
 pytest-mpl
+pytest-benchmark
+markupsafe==2.0.1
```

Comment:
is this needed with the jinja thing fixed?

Reply:
I do not think so, I'll remove